### PR TITLE
declare types of variants

### PR DESCRIPTION
**Pull request by niteshMeena1**

### CODE REVIEW

File: src/components/Chips/index.tsx [hunk 1]
```diff
@@ -1,89 +1,107 @@
...
+  variant?: ChipsTypes;
+  children?: ReactNode;
+};
 
 export const Chips = forwardRef<HTMLDivElement, ChipsProps>(
-  ({ variant = "default", size, colorscheme, className, addRightIcon = false, addLeftIcon = false,children, ...props }, ref) => {
+  ({ variant = ChipsTypes.default, size, colorscheme, className, addRightIcon = false, addLeftIcon = false, children, ...props }, ref) => {
+    let hasText = false;
+
+    switch (true) {
```
Comment (deeporiontechnology) [inline src/components/Chips/index.tsx:77]: what is this ??

File: src/components/SingleSpliceChart/index.stories.tsx [hunk 2]
```diff
@@ -16,7 +17,7 @@ type Story = StoryObj<typeof meta>;
 
 export const Default: Story = {
   args: {
-    variant: "single_spline_area_chart",
+    variant: singleSpliceType.singleSpliceChart,
     value1: [600, 800, 1100, 500, 1000, 700, 800],
     days: ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"],
```
Comment (deeporiontechnology) [inline src/components/SingleSpliceChart/index.stories.tsx:22]: create enum or object

File: src/components/SingleSpliceChart/index.tsx [hunk 3]
```diff
@@ -16,27 +17,27 @@ const singleSplineStyle = cva(
...
         colorscheme: "primary",
         className: "text-[#DC2626] ",
       },
     ],
     defaultVariants: {
-      variant: "single_spline_area_chart",
-      size: "md",
+      variant:singleSpliceType.singleSpliceChart,
+      size: singleSpliceSize.medium,
       colorscheme: "primary",
```
Comment (deeporiontechnology) [inline src/components/SingleSpliceChart/index.tsx:41]: defined the name in enum for primary secondary etc etc

File: src/components/Chips/index.tsx (hunk 1 above)
Comment (niteshMeena1) [reply src/components/Chips/index.tsx:77]: on which line that switch case ya variant type ?

File: src/components/SingleSpliceChart/index.stories.tsx [hunk 4]
```diff
@@ -16,7 +17,7 @@ type Story = StoryObj<typeof meta>;
 
 export const Default: Story = {
   args: {
-    variant: "single_spline_area_chart",
+    variant: singleSpliceType.singleSpliceChart,
     value1: [600, 800, 1100, 500, 1000, 700, 800],
```
[the remark this answers is not in the document]
Comment (niteshMeena1) [reply src/components/SingleSpliceChart/index.stories.tsx:21]: ok

File: src/components/SingleSpliceChart/index.stories.tsx (hunk 2 above)
Comment (niteshMeena1) [reply src/components/SingleSpliceChart/index.stories.tsx:22]: ok

File: src/components/SingleSpliceChart/index.tsx (hunk 3 above)
Comment (niteshMeena1) [reply src/components/SingleSpliceChart/index.tsx:41]: ok

File: src/components/Charts/ChartsTypes.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+export enum ChartsType {
+    doubleSplineAreaChart = "Double Spline Area Chart"
+}
+
+ export enum ChartValueType {
+    Valuex = "ValueX",
```
Comment (deeporiontechnology) [inline src/components/Charts/ChartsTypes.tsx:6]: xAxisValue
yAxisValue
xAxisLabel
yAxisLabel

Don't add days in this enum

Comment (niteshMeena1) [reply src/components/Charts/ChartsTypes.tsx:6]: got it.

Comment (niteshMeena1) [reply src/components/Charts/ChartsTypes.tsx:6]: resolved .

**Pull request merged**